custom/lib/User.php

```diff
@@ -263,4 +267,15 @@ public function revokeRefreshTokens(string $userId): void
             throw PassageError::fromApiException($e);
         }
     }
+
+    private function castUserInfoToPassageUser(UserInfo $userInfo): PassageUser
```

Comment:
duplicated just to keep this function private and to not have to create a whole base class to inherit from just for this.